contrib/syslog-ng-debun

```diff
@@ -870,6 +871,14 @@ fhs_set_unix () {
 	:
 }
 
+acquire_running_syslog_config() {
+
+	if ${syslogngctlbin} config ; then	
```

Comment:
What is the purpose of this line? Is this how we check if syslog-ng is running?

Reply:
@furiel : This is just to check ```syslog-ng-ctl``` is present on the disk and it could fetch config from currently running syslog-ng process.